src/main/resources/project/parameterForms/ec_parameterForm-CheckPageStatus.xml

```diff
@@ -17,6 +17,14 @@
 -->
 <editor>
   <help>/commander/pages/@PLUGIN_NAME@/@PLUGIN_KEY@_help?s=Administration&amp;ss=Plugins#CheckPageStatus</help>
+  <formElement>
+    <type>entry</type>
+    <label>Configuration Name:</label>
+    <property>configname</property>
+    <required>0</required>
+    <documentation>Name of the configuration to be used. URL, port and credentials are retrieved from the given configuration. To view or create a new configuration, go to the Administration -> Plugins tab, and select 'Configure' action for @PLUGIN_KEY@ plugin.</documentation>
+    <htmlDocumentation>Name of the configuration to be used. URL, port and credentials are retrieved from the given configuration.<br/><br/>A Configuration defines connection details and can be created by going to plugin <a style="text-decoration: none !important; border-bottom-style: dashed; border-bottom-width: thin; font-size: inherit; color: inherit; font-family: inherit; border-color: #d8d8d8; border-spacing: 2px;" target="_blank" href="/commander/pages/@PLUGIN_NAME@/configurations">configuration page</a>.</htmlDocumentation>
```

Comment:
Lets define the style in the pluginhelp.css https://github.com/electric-cloud/EC-WebLogic/blob/master/htdocs/pluginhelp.css and refer to it here. Lets also update the base pluginhelp.css if necessary.

Reply:
Since this change applies to other plugins also, this can be done separately as part of implementing the best practices across the plugins. Rest of the changes in this PR look good to me.